pyproject.toml

```diff
@@ -0,0 +1,232 @@
+[project]
+name = "chickadee"
+version = "0.6.1"
+description = "A Web Processing Service for PCIC's ClimDown package."
+authors = [
+  { name = "Nikola Rados" },
+  { name = "Sangwon Lim" },
+  { name = "Eric Yvorchuk" },
+  { name = "Cairo Sanders" },
+  { name = "Quintin Sparks", email = "quintins@uvic.ca" },
+]
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Intended Audience :: Developers",
+  "Intended Audience :: Science/Research",
+  "Operating System :: MacOS :: MacOS X",
+  "Operating System :: POSIX",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Topic :: Scientific/Engineering :: Atmospheric Science",
+  "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+]
+
+license = { text = "GNU General Public License v3" }
+readme = "README.md"
+requires-python = ">=3.11, <=3.14"
+dependencies = [
+  "rpy2==3.3.6",
+
+  "click>=8.2.1,<9.0.0",
+  "gunicorn>=23.0.0,<24.0.0",
+  "jinja2>=3.1.6,<4.0.0",
+  "nchelpers>=5.5.12,<6.0.0",
+  "netcdf4>=1.7.2,<2.0.0",
+  "poetry>=2.1.3,<3.0.0",
+  "psutil>=7.0.0,<8.0.0",
+  "pywps>=4.6.0,<5.0.0",
+  "setuptools>=80.9.0,<81.0.0",
+  "werkzeug>=3.1.3,<4.0.0",
+  "wps-tools>=2.1.4,<3.0.0",
+
+
+]
+
+[project.optional-dependencies]
+dev = [
+  "birdhouse_birdy>=0.8.7,<1.0.0",
+  "black>=25.1.0,<26.0.0",
+  "bumpversion>=0.6.0,<1.0.0",
+  "cruft>=2.16.0,<3.0.0",
+  "flake8>=7.2.0,<8.0.0",
+  "geopandas>=1.1.0,<2.0.0",
+  "ipyleaflet>=0.19.2,<1.0.0",
+  "ipython>=9.3.0,<10.0.0",
+  "ipywidgets>=8.1.7,<9.0.0",
+  "jupyterlab>=4.4.3,<5.0.0",
+  "matplotlib>=3.10.3,<4.0.0",
+  "nbconvert>=7.16.6,<8.0.0",
+  "nbsphinx>=0.9.6,<1.0.0",
+  "nbval>=0.11.0,<1.0.0",
+  "pyproj>=3.7.1,<4.0.0",
+  "pytest>=8.4.0,<9.0.0",
+  "pytest_flake8>=1.3.0,<2.0.0",
+  "pytest_notebook>=0.10.0,<1.0.0",
+  "requests_html>=0.10.0,<1.0.0",
+  "shapely>=2.1.1,<3.0.0",
+  "sphinx>=8.2.3,<9.0.0",
+  "twine>=6.1.0,<7.0.0",
+  "xarray>=2025.4.0,<2026.0.0",
+  "xclim>=0.57.0,<1.0.0",
+]
```

Comment:
quail's pyproject.toml had a note about updating optional dependencies to problems further down the line; does that apply here as well, and if so would it be useful to note it here too?